### DISCUSSION TITLE
Image not displayed in the webpage

### DISCUSSION THREAD

**Discussion by RenukaMenon** [mkdocs/mkdocs#2583] [Q&A] [answered]
Hi,
I'm playing with mkdocs. I made a markdown file in vs code. Included images. They were displayed in the preview, but not in the dev-server. Kindly provide suggestions.

I have used this template for inserting image:  
![image](https://user-images.githubusercontent.com/91125098/134154380-25d3fcd3-2c38-40a1-9801-fcdf5422e1e5.png)

**Comment by RenukaMenon** [comment]
Hi,
I found out the error here. 
This discussion can be closed.

**Reply by ultrabug** [reply]
Please mark how you solved this, and mark it as an anwser

**Comment by RenukaMenon** [comment, marked as answer]
The error was in the folder name. It was "Images". What I entered in the command was "images".
